sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -0,0 +1,480 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+# pylint: disable=protected-access
+
+from re import findall
```

Comment:
Coding style; please just do an `import re`. It is much clearer when looking at the code below that `findall` is actually `re.findall` and not a function in the `_models.py` module.